assembly/src/parser/grammar.lalrpop

```diff
@@ -192,22 +193,36 @@ extern {
         "u32xor" => Token::U32Xor,
         "while" => Token::While,
         "xor" => Token::Xor,
+        "@" => Token::At,
         "!" => Token::Bang,
         "::" => Token::ColonColon,
         "." => Token::Dot,
+        "," => Token::Comma,
         "=" => Token::Equal,
         "(" => Token::Lparen,
+        "[" => Token::Lbracket,
         "-" => Token::Minus,
         "+" => Token::Plus,
         "//" => Token::SlashSlash,
         "/" => Token::Slash,
         "*" => Token::Star,
         ")" => Token::Rparen,
+        "]" => Token::Rbracket,
```

Comment:
Do we still need to define `Lbracket` and `Rbracket`? Doesn't seem like we use them anywhere any more. But also, maybe it doesn't hurt to have them?

Reply:
I considered removing them, as they aren't currently used, but they may come in handy in future syntax extensions, and since they are exceedingly common punctuation characters and this is just adding support for those characters to the lexer/tokenizer, there is basically zero reason not to support them there now that the work has been done. Until they are added to an actual grammar rule, their presence as tokens doesn't actually impact parsing in any way.